lib/election.js

```diff
@@ -771,7 +771,24 @@ function _getManifestFromPeer(
     },
     manifest: callback => api._client.getManifest(
       manifestHash, peerId, callback),
-    sync: ['hashes', 'manifest', (results, callback) => {
+    checkMajority: ['manifest', (results, callback) => {
+      const manifest = results.manifest;
+      const blockHeight = manifest.blockHeight;
+      const manifestItemCount = manifest.item.length;
+      api.getBlockElectors(ledgerNode, blockHeight, (err, result) => {
```

Comment:
I can see us wanting to just pass the `electors` all the way through to here in the future (we do have them but it's several calls up the stack before `_gossipWith`). But... we can leave that to another day to optimize. Perhaps another TODO here above `getBlockElectors` that says: "TODO: consider passing `electors` through to optimize away this look up`"